components/tabs/tabs.js

```diff
@@ -0,0 +1,388 @@
+				margin-left: -3px;
+				padding-left: 3px;
+				position: relative;
+				white-space: nowrap;
+				-webkit-transition: max-width 200ms ease-in;
+				transition: max-width 200ms ease-in;
+			}
+			.d2l-tabs-container-ext {
+				flex: none;
+				padding-left: 4px;
+			}
+			:host([dir="rtl"]) .d2l-tabs-container-ext {
+				padding-left: 0;
+				padding-right: 4px;
+			}
+			.d2l-tabs-container-list {
+				display: block;
+				white-space: nowrap;
+				-webkit-transition: transform 200ms ease-out;
+				transition: transform 200ms ease-out;
+			}
+			.d2l-tabs-focus-start,
+			.d2l-tabs-focus-end {
+				position: absolute;
+				left: 0;
+			}
+			.d2l-tabs-scroll-previous-container,
+			.d2l-tabs-scroll-next-container {
+				background-color: var(--d2l-tabs-background-color);
+				box-shadow: 0 0 12px 18px var(--d2l-tabs-background-color);
+				display: none;
+				height: 100%;
+				position: absolute;
+				top: 0;
+				z-index: 1;
+			}
+			.d2l-tabs-scroll-previous-container {
+				left: 0;
+				margin-left: 4px;
+			}
+			:host([dir="rtl"]) .d2l-tabs-scroll-previous-container {
+				left: auto;
+				margin-left: 0;
+				margin-right: 4px;
+				right: 0;
+			}
+			.d2l-tabs-container[allow-scroll-previous] > .d2l-tabs-scroll-previous-container {
+				display: inline-block;
+			}
+			.d2l-tabs-scroll-next-container {
+				margin-right: 4px;
+				right: 0;
+			}
+			:host([dir="rtl"]) .d2l-tabs-scroll-next-container {
+				left: 0;
+				margin-left: 4px;
+				margin-right: 0;
+				right: auto;
+			}
+			.d2l-tabs-container[allow-scroll-next] > .d2l-tabs-scroll-next-container {
+				display: inline-block;
+			}
+			.d2l-tabs-scroll-button {
+				background-color: transparent;
+				border: 1px solid transparent;
+				border-radius: 15px;
+				box-shadow: 0 0 0 4px rgba(0, 0, 0, 0);
+				box-sizing: border-box;
+				cursor: pointer;
+				display: inline-block;
+				height: 30px;
+				margin: 0;
+				outline: none;
+				padding: 0;
+				transform: translateY(10px);
+				-webkit-transition: box-shadow 0.2s;
+				transition: box-shadow 0.2s;
+				width: 30px;
+			}
+			.d2l-tabs-scroll-button[disabled] {
+				cursor: default;
+				opacity: 0.5;
+			}
+			.d2l-tabs-scroll-button::-moz-focus-inner {
+				border: 0;
+			}
+			.d2l-tabs-scroll-button[disabled]:hover,
+			.d2l-tabs-scroll-button[disabled]:focus {
+				background-color: transparent;
+			}
+			.d2l-tabs-scroll-button:hover,
+			.d2l-tabs-scroll-button:focus {
+				background-color: var(--d2l-color-gypsum);
+			}
+			.d2l-tabs-scroll-button:focus {
+				border-color: var(--d2l-color-celestine);
+			}
+
+			@media (prefers-reduced-motion: reduce) {
+
+			}
+		`];
+	}
+
+	static get resources() {
+		return {
+			'ar': { 'scroll.previous': 'التمرير إلى اليسار', 'scroll.next':'التمرير إلى اليمين' },
+			'en': { 'scroll.previous': 'Scroll Backward', 'scroll.next': 'Scroll Forward' },
+			'es': { 'scroll.previous': 'Desplácese a la izquierda', 'scroll.next': 'Desplácese a la derecha' },
```

Comment:
We should get translations for German and Danish, as they're languages we now support.